ubuntu-headless-qt/include/05_create_wic.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# --------------------------------------------------------------------------#
+# function create_wic use to create wic from ubuntu rootfs.
+# function create_wic contain 5 steps:
+# Step 1: Create blank *.wic
+# Step 2: Create 2 partition using fdisk
+# Step 3: Format partition
+# Step 4: Mount and copy data
+# Step 5: Done
+# --------------------------------------------------------------------------#
+
+function create_wic() {
+    sudo apt-get update
+    sudo apt-get install -y parted multipath-tools kpartx dosfstools e2fsprogs
+
+    ROOTFS_DIR="./rootfs"
+    OUTPUT_WIC="ubuntu-image-qt-rzpi.wic"
+
+    # Define Size
+    BOOT_SIZE_MB=200
```

Comment:
Should we move these variables into a "configuration" file? So Users can set them to whatever they want instead of hardcoding.

Reply:
The configuration file could be "config.ini" and it should be located at the same level as the main script

Reply:
About the configuration file, I will use config.sh so main script can `source config.sh` and get those variables easier